tests/core/pm-module/conftest.py

```diff
@@ -0,0 +1,259 @@
+import json
+import logging
+import pytest
+import sys
+
+from eth_tester import (
+    EthereumTester,
+    PyEVMBackend,
+)
+from eth_utils import (
+    function_abi_to_4byte_selector,
+    to_canonical_address,
+)
+from ethpm import (
+    ASSETS_DIR,
+    Package,
+)
+from ethpm.contract import (
+    LinkableContract,
+)
+from pytest_ethereum import (
+    linker as l,
+)
+from pytest_ethereum.deployer import (
+    Deployer,
+)
+
+from web3 import Web3
+from web3.pm import (
+    SolidityReferenceRegistry,
+    VyperReferenceRegistry,
+)
+
+VY_PACKAGE_ID_1 = b'\xd0Y\xe8\xa6\xeaZ\x8b\xbf\x8d\xd0\x97\xa7\xb8\x92#\x16\xdc\xb7\xf0$\xe8"\x0bV\xd3\xc9\xe7\x18\x8ajv@'  # noqa: E501
```

Comment:
It *might* be nice to represent these as their hex values like:

```python
VY_PACKAGE_ID_1 = to_bytes(hexstr='0xd0...')  # noqa: E501
...
```

As those values are easier to visually parse.